Packs/BeyondTrust_Password_Safe/README.md

```diff
@@ -1,2 +1,88 @@
 # Beyond Trust Password Safe
-Unified password and session management for seamless accountability and control over privileged accounts.
\ No newline at end of file
+Unified password and session management for seamless accountability and control over privileged accounts.
+
+<~XSIAM>
+
+## Cortex XSIAM SIEM Content
+
+This pack includes Cortex XSIAM SIEM content for parsing and modeling the syslog events that are forwarded from BeyondTrust Beyond Safe. 
```

Comment:
Should this be BeyondTrust **Password** Safe?

Reply:
I agree, BeyondTrust seems correct in contrast to Beyond Trust, I'll fix it accordingly.